bouncer/shared/create_and_delete_multiple_orders.ts

```diff
@@ -1,54 +1,97 @@
+import { deposits } from './setup_swaps';
 
-export async function createAndDeleteMultipleOrders(numberOfOrders: number) {
+const LP: string = '//LP_3';
+
+async function countOpenOrders(
+  baseAsset: string,
+  quoteAsset: string,
+  lp: string,
+  chainflip: ApiPromise,
+) {
+  const orders = await chainflip.rpc('cf_pool_orders', baseAsset, quoteAsset, lp);
+  if (!orders) {
+    throw Error('Rpc cf_pool_orders returned undefined');
+  }
+  let openOrders = 0;
+
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.asks.length || 0;
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.bids.length || 0;
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.range_orders.length || 0;
+
+  return openOrders;
+}
+
+export async function createAndDeleteMultipleOrders(numberOfLimitOrders: number) {
   console.log(`=== cancel_orders_batch test ===`);
   await using chainflip = await getChainflipApi();
 
   const keyring = new Keyring({ type: 'sr25519' });
   keyring.setSS58Format(2112);
-  const lpUri = process.env.LP_URI || '//LP_1';
+  const lpUri = process.env.LP_URI || LP;
   const lp = keyring.createFromUri(lpUri);
 
+  await Promise.all([
+    // provide liquidity to LP_3
+    depositLiquidity('Usdc', 10000, false, LP),
+    depositLiquidity('Eth', deposits.get('Eth')!, false, LP),
+    depositLiquidity('Dot', deposits.get('Dot')!, false, LP),
+    depositLiquidity('Btc', deposits.get('Btc')!, false, LP),
+    depositLiquidity('Flip', deposits.get('Flip')!, false, LP),
+    depositLiquidity('Usdt', deposits.get('Usdt')!, false, LP),
+    depositLiquidity('ArbEth', deposits.get('ArbEth')!, false, LP),
+    depositLiquidity('ArbUsdc', deposits.get('ArbUsdc')!, false, LP),
+    depositLiquidity('Sol', deposits.get('Sol')!, false, LP),
+    depositLiquidity('SolUsdc', deposits.get('SolUsdc')!, false, LP),
+  ]);
+
   // create a series of limit_order and save their info to delete them later on
   const promises = [];
   const orderToDelete: {
     Limit?: { base_asset: string; quote_asset: string; side: string; id: number };
     Range?: { base_asset: string; quote_asset: string; id: number };
   }[] = [];
   let i = 0;
-  while (i < numberOfOrders) {
-    promises.push(limitOrder('Btc', 0.00000001, i, i));
+  while (i < numberOfLimitOrders) {
+    promises.push(limitOrder('Btc', 0.00000001, i, i, LP));
     orderToDelete.push({ Limit: { base_asset: 'BTC', quote_asset: 'USDC', side: 'sell', id: i } });
     i++;
   }
+  i = 0;
+  while (i < numberOfLimitOrders) {
+    promises.push(limitOrder('Eth', 0.000000000000000001, i, i, LP));
+    orderToDelete.push({ Limit: { base_asset: 'ETH', quote_asset: 'USDC', side: 'sell', id: i } });
+    i++;
+  }
+
+  promises.push(rangeOrder('Btc', 0.1, LP));
+  orderToDelete.push({
+    Range: { base_asset: 'BTC', quote_asset: 'USDC', id: 0 },
```

Comment:
We could instead get the await on the order creations and get the OrderId, rather than rely on it always being 0. If we run this test again it'll fail because of this.

We can likely do it such that we can still Promise.all() on all the orders at the same time, and then we can get the order ids of the respective orders to place in ordersToDelete

Reply:
This is not true, the test pass every time since rangeOrder always create an order with id = 0;
The test won't pass only if the LP is used to other tests, in which it is used to create more orders with an id that doesn't get deleted by this test, but the whole point was to create this new lp specifically for this test so that we could run it in parallel hence I don't think this is a problem.

Reply:
My thinking is beyond just this test - if we wanted to right something like a "chaos mode" that runs a bunch of bouncer tests randomly one after the other, then we would want to be able to run it twice without assuming an order id (applies to limit orders too ofc not just here)

Reply:
yeah but what I am saying is that this test always create orders with id starting from 0 to numLimitOrders (for limit orders) and id = 0 for range orders and deletes them no matter what.
It works as many times in a row you want. 
For limit orders this is not required since we are passing the orderId to the function, hence we are sure it will always delete the order it creates, I could do the same for the rangeOrder function if we want to be sure to delete the same orders we create (I just used 0 since it was hardcoded in the rangeOrder function).